### DISCUSSION TITLE
PagerDuty integration - resolve the PD incident too

### DISCUSSION THREAD

**Discussion by chadcancode** [incidentbot/incidentbot#72] [Ideas]
It looks like incident-bot can send a new PagerDuty alert when an incident is created. This is great. What would be cool too is if incident-bot would resolve the PD alert.

**Comment by echoboomer** [comment]
@chadcancode I agree, this can get worked in fairly easily.

**Reply by echoboomer** [reply]
This has been implemented in the latest release.
